fiftyone/utils/torch.py

```
@@ -2605,6 +2667,245 @@ def _parse_inputs(
+            Tensor of processed frames ready for model
+        """
+        processed_frames = []
+        if frame_dicts is None:
+            frame_dicts = [{}] * len(raw_frames)
+        for frame, frame_dict in zip(raw_frames, frame_dicts, strict=True):
+            if self.video_get_item is not None:
+                frame_dict["frame"] = frame
+                frame_input = self.video_get_item(frame_dict)
+            else:
+                if self.use_numpy is False:
+                    frame = Image.fromarray(frame)
```

Ruff (0.15.12)

[warning] `for` loop variable `frame` overwritten by assignment target

(PLW2901)